docs/usage/import-automated.md

```diff
@@ -1,7 +1,89 @@
+
+In summary, automated imports through connectors empower OpenCTI users with a scalable, efficient, and customizable mechanism for data ingestion, ensuring that the platform remains enriched with the latest and most relevant intelligence.
+
+
+## Native automated import
+
+In OpenCTI, the "Data > Ingestion" section provides users with built-in functions for automated data import. These functions are designed for specific purposes and can be configured to seamlessly ingest data into the platform. Here, we'll explore the configuration process for the three built-in functions: Live Streams, TAXII Feeds, and RSS Feeds.
+
+### Live streams
+
+Live Streams enable users to consume data from another OpenCTI platform, fostering collaborative intelligence sharing. Here's a step-by-step guide to configure Live streams synchroniser:
+
+1. Remote OpenCTI URL: Provide the URL of the remote OpenCTI platform (e.g., `https://[domain]`; don't include the path).
+2. Remote OpenCTI token: Provide the user token. An administrator from the remote platform must supply this token, and the associated user must have the "Access data sharing" privilege.
+3. After filling in the URL and user token, validate the configuration.
+4. Once validated, select a live stream to which you have access.
+
+![Live stream configuration](assets/live-stream-configuration.png)
+
+Additional configuration options:
+
+- User responsible for data creation: Define the user responsible for creating data received from this stream. Best practice is to dedicate one user per source for organizational clarity. Please [see here](../deployment/connectors.md#connector-token-section) for more information on this good practice.
+- Starting synchronization: Specify the date of the oldest data to retrieve. Leave the field empty to import everything.
+- Take deletions into account: Enable this option to delete data from your platform if it was deleted on the providing stream. (Note: Data won't be deleted if another source has imported it previously.)
+- Verify SSL certificate: Check the validity of the certificate of the domain hosting the remote platform.
+- Avoid dependencies resolution: Import only entities without their relationships. For instance, if the stream shares malware, all the malware's relationships will be retrieved by default. This option enables you to choose not to recover them.
+- Use perfect synchronization: This option is specifically for synchronizing two platforms. If an imported entity already exists on the platform, the one from the stream will overwrite it.
+
+![Live stream additional configuration](assets/live-stream-additional-configuration.png)
+
+### TAXII feeds
+
+TAXII Feeds in OpenCTI provide a robust mechanism for ingesting TAXII collections from TAXII servers or other OpenCTI instances. Configuring TAXII ingester involves specifying essential details to seamlessly integrate threat intelligence data. Here's a step-by-step guide to configure TAXII ingesters:
```

Comment:
TAXII ingestors ?

Reply:
Perhaps you're right, in which case it needs to be changed in the documentation and in the platform.